### ISSUE TITLE
Icon is not being set on electron.exe properly

### ISSUE THREAD

**Issue by Alex313031** [microsoft/vscode-gulp-electron#16]
I have an extremely annoying issue.

I have forks of vscode and vscodium, and they both have this issue.

Everything builds correctly, and the app runs correctly, but the icon of electron.exe is not set properly.

See my issues posted in both repos.

https://github.com/microsoft/vscode/issues/191077
and
https://github.com/VSCodium/vscodium/issues/1571

Attached screenshot
![262693182-da8f1d9d-2afe-45ce-806a-7ba7c82fd223](https://github.com/microsoft/vscode-gulp-electron/assets/45863095/a6151a69-bb88-4d06-8910-8be35b9fe45f)

I think it has to do with this, as there are only two places in the source where code.ico is used. One in electron.ts, and one in the gulpfile config which sets winIcon: resources/win32/code.ico. I see that this package is using rcedit to replace the icon file in electron.exe, and then renaming electron.exe to the product name.

This issue has been persistent since vscode 1.79.0

**Comment by Alex313031** [comment]
@deepak1556 @joaomoreno It seems the issue lies with the fact you are using a 5 year old rcedit (0.3.0), while vscode is using 1.1.2. Running tests with using rcedit 0.3.0 to modify notepad.exe causes node errors, using 1.1.0 or above changes the metadata correctly.

**Comment by deepak1556** [comment]
Thanks for investigating the issue, feel free to send a PR for updating rcedit package version.

**Comment by Alex313031** [comment]
@deepak1556 YUP I WAS SO FRUSTRATED. I have been working on this for over a week, Because I cannot ship a Codium release to my users with the icon screwed up.

I made a fork of this repo, and the only thing I changed was the rcedit version. I then changed the package.json of VSCode to use my fork, and the icon is correct now!! Im so happy, I will send a PR right away.

@gz83 look at this, they responded! Now we can make fresh Codium releases!

**Comment by joaomoreno** [comment]
As indicated in README.md: this library is deprecated and not meant for use outside of VS Code.
